axelrod/tournament.py

```diff
@@ -1,11 +1,12 @@
 """Recreate Axelrod's tournament."""
 
 import inspect
-import itertools
+import numpy
```

Comment:
It's not just creating an empty matrix, it's creating an array filled with zeros. Plus, having that array allows indexing that replaces cumbersome nested loops. Nonetheless, it can be replaced easily (see below) and the performance gain is negligible at this point.